src/scripts/sonar.sh

```diff
@@ -33,7 +33,7 @@ function detect_maven {
     return
   fi
   
-  if hash mvn 2>/dev/null; then
+  if hash mvn 2>/dev/null && [ -f "${PROJECT_DIR}/pom.xml" ] ; then
```

Comment:
```suggestion
  if hash mvn 2>/dev/null && [ -f "pom.xml" ] ; then
```

per uniformità con l'if precedente

forse andrebbe reso più ovvio che il comando deve essere eseguito dalla cartella del progetto maven (potrebbe non essere la root del repo), non so bene come/quando

Reply:
`PROJECT_DIR` viene valorizzata da un parametro dello step `test_sonar` (di default ha valore `.`), in questo modo chi esegue il comando `test_sonar` deve solo indicare la folder del progetto (spostandosi lì, in un certo senso)

Reply:
aaaah ok, pensavo fosse la root del repo